application/apps/indexer/session/src/state/indexes/tests_map_performance.rs

```diff
@@ -0,0 +1,106 @@
+use super::{map::Map, nature::Nature};
```

Comment:
performance tests must not be part of the unit tests. 
1. needs to be built as `release`
2. takes way too much time

Reply:
commented out